test/benchmarks/001-transaction-and-span-no-stack-trace.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+/* eslint-disable no-unused-vars, no-undef */
+
+const bench = require('./utils/bench')
+
+bench('transaction-and-span-no-stack-trace', {
+  agentConf: {
+    captureSpanStackTraces: false
+  },
+  setup () {
+    var agent = this.benchmark.agent
```

Comment:
I guess this gets into the `fn(...)` scope below by benchmark.js generating code from the contents of the input functions and eval'ing that? 🤔 

Seems a bit confusing and looks like dead code at first glance. Also, I feel like the linter would likely complain about this sort of thing.

Reply:
Correct. Which is also why I have to use `var` instead of `const` as those are not hoisted in the generated code where it happens to wrap the `setup` in a try-catch. To get the linter to behave, I've added a `eslint-disable` comment at the top of the file.

Check out this example to see how the code compilation works: https://benchmarkjs.com/docs#prototype_setup

Reply:
Gotcha. 👍